src/javascript/Validation/validation.spec.js

```diff
@@ -373,6 +373,122 @@ describe('validation', () => {
         });
     });
 
+    describe('email', () => {
+        it('should trigger error when is not valid email', () => {
```

Comment:
```suggestion
        it('should trigger error when is not valid pattern', () => {
```

Je ne le fais pas pour la suite, t'as compris l'idée :-)